gcb_web_auth/utils.py

```diff
@@ -6,11 +6,26 @@
 from gcb_web_auth.models import OAuthToken, OAuthService, DukeDSAPIToken, DDSEndpoint
 from jwt import decode, InvalidTokenError
 from django.core.exceptions import ObjectDoesNotExist
+from gcb_web_auth.exceptions import *
 import logging
 
 logger = logging.getLogger(__name__)
 
 
+def get_default_dds_endpoint():
+    endpoint = DDSEndpoint.objects.first()
+    if not endpoint:
+        raise DDSConfigurationException('No DDSEndpoint is configured')
+    return endpoint
+
+
+def get_default_oauth_service():
```

Comment:
It would be nice to have comments for `get_default_dds_endpoint` and `get_default_oauth_service`.